pytorch_pfn_extras/logging.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import os
+
+_logger_name = 'ppe'
+_logger = None
+
+
+def configure_logging(filename=None, level='ERROR'):
```

Comment:
This should be a private method and without arguments as users cannot change the filename / level after importing PPE.

```suggestion
def _configure_logging():
```